Regression on ROS 2 Buildfarm for Rolling

`tuw_geometry 0.1.2` is failing to build on the buildfarm for Rolling. ' The package was likely linking against ament_cmake_ros transitively but we'll need to be explicit after https://github.com/ros/rosdistro/pull/44971

This is blocking the [announced Rolling sync](https://discourse.ros.org/t/preparing-for-rolling-sync-2025-04-11/43166). We will proceed with the sync which will remove these binaries from Rolling but they will be added once the build is fixed and a new release is bloomed.

See detailed logs: https://build.ros2.org/view/Rbin_uN64/job/Rbin_uN64__tuw_geometry__ubuntu_noble_amd64__binary/61/console

```
23:15:23 CMake Error at CMakeLists.txt:12 (find_package):
23:15:23   By not providing "Findament_cmake_ros.cmake" in CMAKE_MODULE_PATH this
23:15:23   project has asked CMake to find a package configuration file provided by
23:15:23   "ament_cmake_ros", but CMake did not find one.
23:15:23 
23:15:23   Could not find a package configuration file provided by "ament_cmake_ros"
23:15:23   with any of the following names:
23:15:23 
23:15:23     ament_cmake_rosConfig.cmake
23:15:23     ament_cmake_ros-config.cmake
23:15:23 
23:15:23   Add the installation prefix of "ament_cmake_ros" to CMAKE_PREFIX_PATH or
23:15:23   set "ament_cmake_ros_DIR" to a directory containing one of the above files.
23:15:23   If "ament_cmake_ros" provides a separate development package or SDK, be
23:15:23   sure it has been installed.
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,6 +10,7 @@ include_directories(include)
 # find dependencies
 find_package(ament_cmake REQUIRED)
 find_package(ament_cmake_ros REQUIRED)
+find_package(gtest_vendor REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(rclcpp_components REQUIRED)
 find_package(std_msgs REQUIRED)

